doc/Language/5to6-perlvar.pod6

```diff
@@ -19,13 +19,11 @@ please see the Raku documentation for each of them.
 
 =head1 SPECIAL VARIABLES
 
-X<|Special Variables (Perl)>
+X<|Foreign,Special Variables (Perl)>
 
 =head2 General variables
 
-=head3 $ARG, $_
-
-X<|$_ (Perl)>
+=head3 X<$ARG, $_|Foreign,$_ (Perl)>
```

Comment:
You're dropping `$ARG` as an indexing term.

Reply:
Actually, not dropping, but not including where it wasn't included before, but anyway.

Reply:
>You're dropping $ARG as an indexing term.

But it never was indexed, so it's equivalent, the text is `$ARG, $_` and `$_ (Perl)` is indexed, the same in the updated version.

Reply:
Fair enough. Still, choosing one over the other is arbitrary and might not have been the best call initially.